.platform.app.yaml

```
@@ -0,0 +1,99 @@
+name: sylius
+
+type: php:8.1
```

Build flavor should be specified at the top.

```suggestion
type: php:8.1
build:
    flavor: none
```

---

Overall placement feels weird, I personally would go for

- type
- build
- runtime
- variables
- dependencies
- hooks
- relationships
- disk
- mounts
- web
- workers
- cron

or last three as
- workers
- cron
- web

---

I will leave it for now as it is similar to config provided for the template by platform.sh 🤔 https://github.com/platformsh-templates/sylius/blob/master/.platform.app.yaml